plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/format/SQLFormatterConfiguration.java

```diff
@@ -106,8 +107,14 @@ public void setSourceEncoding(@NotNull String sourceEncoding) {
         this.sourceEncoding = sourceEncoding;
     }
 
-    public boolean isFunction(String name) {
-        return syntaxManager.getDialect().getFunctions().contains(name.toUpperCase(Locale.ENGLISH));
+    public boolean isFunction(@NotNull String name) {
+        Set<String> functions = syntaxManager.getDialect().getFunctions();
```

Comment:
It is very slow. We should keep caqche in upper case. 
We may keep original value too